content/2016.09.jedit-trackers-with-kinto-en.md

```diff
@@ -0,0 +1,403 @@
+Title: Getting jEdit Trackers off the Ground using Kinto and Kinto.js
+Date: 2016-09-09
+Lang: en
+Url: jedit-trackers-with-kinto
+Slug: jedit-trackers-with-kinto
+Summary: My experience using Kinto and Kinto.js to browse jEdit tickets offline
+
+# Getting jEdit Trackers off the Ground using Kinto and Kinto.js
+
+This is a guest post by [Eric Le Lay](https://github.com/elelay) on his experience using Kinto.
+
+## Introduction
+
+Do you know [jEdit](http://jedit.org) — *programmer's text editor*?  
+It's a multi-platform text editor, written in java, with a myriad of plugins.
+I've been part of the development team for ten years.
+It's hosted on Sourceforge, with the code base under *subversion*.
+
+This is a labour of love for us, contributing during our free time.
+I don't always have an Internet access on holidays. Or it's via a short Ethernet cable in a dark and overheated room.
+Don't lock me out of the sunny garden! Give me everything I need off-line!
+
+ - java sources and javadoc: install the `openjdk7-doc` and `openjdk7-src` packages;
+ - jEdit and plugins sources: with git-svn I get the code and history;
+ - **bugs and feature requests: oops!**
+ 
+We have 1100 open tickets and 7600 closed. All of that is not available offline. It must end!
```

Comment:
You can change "All of that is not" to "None of that is".

Instead of "It must end", I'd probably write "That needs to stop". But it's strange to talk about ending the non-existence of something, so I'd probably write "That needs to change!"